src/component/utility/export.ts

```diff
@@ -268,6 +268,37 @@ function getBlob(rootRef: HTMLDivElement, elementID: string): BlobObject {
   return { blob, width, height };
 }
 
+function getMoleculesElement(rootRef) {
+  const nmriumViewer: any = (rootRef.getRootNode() as Document).querySelector(
+    `#nmrium-viewer`,
+  );
+
+  const floatingMoleculesGroup = document.createElement('g');
+
+  for (const element of nmriumViewer.querySelectorAll('.draggable-molecule')) {
+    const transform = window
+      .getComputedStyle(element)
+      .getPropertyValue('transform');
+    const matrix = new WebKitCSSMatrix(transform);
```

Comment:
it seems that the standard name for this is `DOMMatrix`: https://developer.mozilla.org/en-US/docs/Web/API/DOMMatrix

Reply:
the WebKitCSSMatrix is an alias to DOMMatrix

Reply:
Yes this is the whole point. To use the preferred name and not an alias.